src/maltihw.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Scanner;
+
+public class maltihw {
+    public static void main(String args[])
+    {
+        Scanner sc = new Scanner(System.in);
+        System.out.print("Enter number: ");
```

Comment:
only asked for print 2 time table so there was no need of scanner.
Git branch  Good
```suggestion
        System.out.print("Enter number: ");
```